### PR TITLE
Update wineskin-winery.rb

### PR DESCRIPTION
**Pull request by Eitot**
• Add zap stanza
• Add minimum OS

### CODE REVIEW

File: Casks/wineskin-winery.rb [hunk 1]
```diff
@@ -7,6 +7,14 @@
...
+  depends_on :macos => '>= :snow_leopard'
 
   app 'Wineskin Winery.app'
+  zap :delete => [
+        '~/Library/Application Support/Wineskin',
+        '~/Library/Preferences/com.urgesoftware.wineskin.wineskin.plist',
+        '~/Library/Caches/com.urgesoftware.wineskin.wineskinwinery',
+        '~/Library/Saved Application State/com.urgesoftware.wineskin.wineskin.savedState'
+      ],
+      :rmdir => '~/Applications/Wineskin' # Won't delete if it contains applications
```
Comment (vitorgalvao) [inline Casks/wineskin-winery.rb]: Comment should be removed. Serves no purpose.

File: Casks/wineskin-winery.rb [hunk 2]
```diff
@@ -7,6 +7,14 @@
   name 'Wineskin Winery'
   homepage 'http://wineskin.urgesoftware.com/'
   license :gpl
+  depends_on :macos => '>= :snow_leopard'
```
Comment (vitorgalvao) [inline Casks/wineskin-winery.rb]: https://github.com/caskroom/homebrew-cask/pull/14326/files#r41712195

File: Casks/wineskin-winery.rb [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
 
   app 'Wineskin Winery.app'
+  zap :delete => [
```
Comment (vitorgalvao) [inline Casks/wineskin-winery.rb]: https://github.com/caskroom/homebrew-cask/pull/14326/files#r41712191

**Comment by Eitot** [comment]
Updated and squashed

**Pull request merged**